### PR TITLE
Calculate implemented

### PR DESCRIPTION
**Pull request by Rogal255**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @Rogal255

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Rogal255 for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+        {
+            return "Division by 0";
+        }
+        num_result = first / second;
+    }
+    else
+    {
+        return "Invalid data";
+    }
+    return std::to_string(num_result);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:31]: You can also return immediately values :) Both option is good. I prefer immediately return, but this is my preference :)

**Review by nauka-programowania-MA — APPROVED** [review]
Perfect! I really enjoy your solutions!

**Pull request closed without merging**